internal/bigqueryclient/client.go

```diff
@@ -1,4 +1,4 @@
-//Package bigqueryclient is responsible for interacting with the BigQuery API.
+// Package bigqueryclient is responsible for interacting with the BigQuery API.
```

Comment:
Looks like gofmt has done something here.. though I think its the formatting that should have been there in the first place.